tests/model_registry/model_catalog/test_huggingface_model_validation.py

```
@@ -0,0 +1,52 @@
+            marks=pytest.mark.install,
+        ),
+    ],
+    indirect=True,
+)
+@pytest.mark.usefixtures("updated_catalog_config_map")
+class TestHuggingFaceModelValidation:
+    """Test HuggingFace model values by comparing values between HF API calls and Model Catalog api call"""
+
+    def test_huggingface_model_metadata(
+        self: Self,
+        updated_catalog_config_map: tuple[ConfigMap, str, str],
```

Ruff (0.14.8)

Unused method argument: `updated_catalog_config_map`

(ARG002)